linux/LinuxProcessList.c

```diff
@@ -900,13 +900,17 @@ static void LinuxProcessList_readCGroupFile(LinuxProcess* process, openat_arg_t
 
    bool changed = !process->cgroup || !String_eq(process->cgroup, output);
 
+   Process_updateFieldWidth(CGROUP, strlen(output));
    free_and_xStrdup(&process->cgroup, output);
 
-   if (!changed)
+   if (!changed) {
+      Process_updateFieldWidth(CCGROUP, strlen(process->cgroup_short));
```

Comment:
`CCGROUP` given by `process->cgroup_short` can be `NULL` in which case this is aliased to `process->cgroup`.